pkg/visor/init.go

```diff
@@ -62,8 +63,18 @@ var (
 	ar vinit.Module
 	// App discovery
 	disc vinit.Module
+	// Stun client
+	sc vinit.Module
+	// SUDPH client
+	sudph vinit.Module
+	// SCTPR client
+	sctpr vinit.Module
+	// SCTP client
+	sctp vinit.Module
 	// Snet (different network types)
 	sn vinit.Module
+	// SnetCheck (check if atleast one network is available)
+	snCheck vinit.Module
```

Comment:
I don't think we need this as a module. The whole point of modules is to run two initialization functions in parallel to gain performance. Basically this module initialization is "is snet already usable?". And anyone that wants snet functionality would sure want it to be usable, so will have to depend on this module instead of snet, and snet and snCheck will have to initialize sequentially all the time.

So I would put the check if snet is ready to be used in snet initialization function

Reply:
All the transport modules depend on the snet module so without it no transports will be initialized.

Reply:
Okay.